### ISSUE TITLE
Nested carousels SCSS issue

### ISSUE THREAD

**Issue by justindmyers** [ganlanyuan/tiny-slider#75]
__Issue description__:  When nesting carousels of different types (gallery/normal) the CSS is selecting all child elements causing visual problems. It's a simple solve. 

Instead of:

```
.tns-gallery {
  ...
  .tns-item {
    ...
  }
}
```

Use the child combinator wherever the child `.tns-child` is used:

```
.tns-gallery {
  ...
  & > .tns-item {
    ...
  }
}
```

**Comment by ganlanyuan** [comment]
Thanks, will add this in next release.

**Comment by ganlanyuan** [comment]
Fixed in [v2.1.8](https://github.com/ganlanyuan/tiny-slider/releases/tag/v2.1.8)
